dotCMS/src/integration-test/java/com/dotmarketing/util/PageModeTest.java

```diff
@@ -111,6 +108,56 @@ public void test_page_mode_does_not_create_session() {
 
     }
 
+    /**
+     * Back end users can have their page mode set to EDIT MODE
+     */
+    @Test
+    public void test_back_end_user_can_be_set_to_LIVE() {
+
+        HttpServletRequest request = backEndRequest();
+
+        assert (PageMode.get(request) == PageMode.PREVIEW_MODE);
+
+        PageMode.setPageMode(request, PageMode.LIVE);
+
+        assert (PageMode.get(request) == PageMode.LIVE);
+
+    }
+
+
+    /**
+     * Testing order of reading PAGE_MODE for a backend user, It should be like this:
+     * 1. request parameter
+     * 2. request attribute
+     * 3. session
+     * 4. default = PREVIEW_MODE
+     * we should respect the request parameter,
+     * then the request attribute, then the session, and finally PREVIEW_MODE if we don't have anything
+     */
+    @Test
+    public void test_order_of_reading_pagemode() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6152219)